scipy/integrate/tests/test_quadpack.py

```diff
@@ -245,9 +245,10 @@ def opts_basic(*args):
             return {'points': [0.2*args[2] + 0.5 + 0.25*args[0]]}
 
         res = nquad(func1, [[0, 1], [-1, 1], [.13, .8], [-.15, 1]],
-                    opts=[opts_basic, {}, {}, {}])
-        assert_quad(res, 1.5267454070738635)
-
+                    opts=[opts_basic, {}, {}, {}], full_output=True)
+        assert_quad(res[:-1], 1.5267454070738635)
+        assert(res[-1]['neval'] == 388962) 
```

Comment:
Use ``assert_`` (``assert`` gets stripped with ``python -O``).

This test is probably fragile, number of evaluations may change on other platforms for example. Maybe just check that ``res[-1]['neval']`` exists, or check for ``<= 400000`` to allow some variation.